core/inventory/inventory.py

```diff
@@ -114,8 +114,7 @@ def __init__(self):
 		# The currently selected slot saved as 0-based index for __slots array
 		self.selectedslot = 0
 
-		# TODO: Old inventory system we discussed... *sigh*
-		# TODO: How one lazy guy can fuck up the entire projects... annoys me
+		# TODO: No Old Inv. System
```

Comment:
Cringe

Reply:
lol heel motiverend om elke keer dat ik in dat bestand zit dat te zien

Reply:
Welke motivatie heb je het over? De motivatie die je stimuleert eens per jaar een regel code te schrijven warmee je somehow de geneva convention weet te overschrijden?

Reply:
stop met berichten sturen en laat me die fixes die jij zo graag wil implementeren

Reply:
haha i am laughter